.github/workflows/update-axe-core.yml

```
@@ -0,0 +1,37 @@
+name: Update axe-core
+
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

```suggestion
          node-version: 18
```
Just because 18 is LTS